docs/en_US/FeatureEngineering/Overview.md

```diff
@@ -241,17 +241,20 @@ print("Pipeline Score: ", pipeline.score(X_train, y_train))
 
 # Benchmark
 
-`Baseline` means without any feature selection, we directly pass the data to LogisticRegression. For this benchmark, we only use 10% data from the train as test data.
-
-| Dataset | Baseline | GradientFeatureSelector | TreeBasedClassifier | #Train | #Feature | 
-| ----------- | ------ | ------ | ------- | ------- | -------- |
-| colon-cancer | 0.7547 | 0.7368 | 0.7223 | 62 | 2,000 |
-| gisette | 0.9725 | 0.89416 | 0.9792 | 6,000 | 5,000 |
-| avazu | 0.8834 | N/A | N/A | 40,428,967 | 1,000,000 |
-| rcv1 | 0.9644 | 0.7333 | 0.9615 | 20,242 | 47,236 |
-| news20.binary | 0.9208 | 0.6870 | 0.9070 | 19,996 | 1,355,191 |
-| real-sim | 0.9681 | 0.7969 | 0.9591 | 72,309 | 20,958 |
-
-The benchmark could be download in [here](https://www.csie.ntu.edu.tw/~cjlin/libsvmtools/datasets/
+`Baseline` means without any feature selection, we directly pass the data to LogisticRegression. For this benchmark, we only use 10% data from the train as test data. For the GradientFeatureSelector, we only take the top20 features.
+
+| Dataset | Baseline | GradientFeatureSelector top20 | GradientFeatureSelector auto |TreeBasedClassifier | #Train | #Feature | 
+| ----------- | ------ | ------ | ------- | ------- | -------- |-------- |
+| colon-cancer | 0.7547 | 0.7368 | 0.5389 | 0.7223 | 62 | 2,000 |
+| gisette | 0.9725 | 0.8942 | 0.9642 |0.9792 | 6,000 | 5,000 |
+| avazu | 0.8834 | N/A   |   N/A   |   N/A | 40,428,967 | 1,000,000 |
+| rcv1 | 0.9644 | 0.7333 | 0.9548 |0.9615 | 20,242 | 47,236 |
+| news20.binary | 0.9208 | 0.8780  | 0.8875 | 0.9070 | 19,996 | 1,355,191 |
+| real-sim | 0.9681 |  0.7969 | 0.9439  |0.9591 | 72,309 | 20,958 |
+| kdd2010  | 0.8826    |  N/A |   N/A   |  N/A  |  8,407,752   | 20,216,830 |
+| webspam  | 0.9848    |  N/A    |   N/A   |  N/A  |  350,000  | 16,609,143 |

```

Comment:
There is no feature selector data for kdd2010 and webspam,  should we remove them?

Reply:
what are the metrics in the table,  the larger the better?